types/wordpress__notices/index.d.ts

```diff
@@ -13,12 +13,46 @@ declare module '@wordpress/data' {
 }
 
 export type Status = 'error' | 'info' | 'success' | 'warning';
+export type WPNoticeType = 'snackbar' | 'default';
```

Comment:
```suggestion
export type NoticeType = 'snackbar' | 'default';
```
as in other place, there is no need really for custom prefix. Is the 'WP' used elsewhere for types here?

Reply:
No, I was basing my changes on the names used in the JSDoc style types in Gutenberg:

https://github.com/WordPress/gutenberg/blob/%40wordpress/notices%403.5.0/packages/notices/src/store/selectors.js#L19-L43

I have reverted it now though if it's not useful.